commons/types/transaction.go

```diff
@@ -52,6 +52,11 @@ type Transaction struct {
 	ToName    string  // Transient
 }
 
+type response struct {
```

Comment:
We already have a `Response` object (types/response.go). This could be `pagingResult` or something?